project/src/com/google/daggerqueryui/page.html

```diff
@@ -25,15 +25,60 @@
     <script src="third-party/js/jquery.min.js"></script>
     <!-- MDB core JavaScript -->
     <script type="text/javascript" src="third-party/js/mdb.min.js"></script>
+    <!-- Bootstrap tooltips -->
+    <script type="text/javascript" src="third-party/js/popper.min.js"></script>
+    <!-- Bootstrap core JavaScript -->
+    <script type="text/javascript" src="third-party/js/bootstrap.min.js"></script>
```

Comment:
nit: How about adding the `defer` attribute to prevent javascript download from blocking the page load? See https://stackoverflow.com/a/24070373

Reply:
Done, thanks ✅